### DISCUSSION TITLE
Source date tracking mode

### DISCUSSION THREAD

**Discussion by JoeCharlesMorgan** [codefori/vscode-ibmi#1637] [Q&A]
Am unable to locate the "Source date tracking mode" button to enable the diff mode to allow search by change date for RPGLE source.  Have opened  connection settings and also performed a search on "diff' but am still unable to locate the button.  As an fyi, this is in reference to #951 where this solution was discussed.  Thanks.

**Comment by SJLennon** [comment]
@JoeCharlesMorgan 
Don't know if this is relevant or not, but in the documentation it says:  
"_For source dates to be enabled, the QCCSID system value must not be 65535._"(https://codefori.github.io/docs/#/pages/developing/sourcedates):

**Comment by worksofliam** [comment]
@JoeCharlesMorgan The date search button is in the bottom right of the status bar when source dates are enabled and diff mode is being used.
